src/main/java/com/exadel/ipromise/dto/PromiseUpdateDto.java

```diff
@@ -0,0 +1,52 @@
+package com.exadel.ipromise.dto;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class PromiseUpdateDto {
```

Comment:
It's not the best idea create separate object if you just want update only one field. For that it's better send "Id" as query and "amount" as parameter - than you avoid redundant convertation. BUT it is just advice, you can leave it as it is.